.github/workflows/Update-Version-and-create-Realeases-PR.yml

```diff
@@ -0,0 +1,50 @@
+name: Update version and create Release's PR Workflow
+
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 'Version name'
+        required: true
+        type: string
```

Comment:
Maybe we want to validate the string with a regex like `v[0-9]\.[0-9]\.[0-9](-[a-zA-Z])?`
https://github.com/orgs/community/discussions/48373#discussioncomment-7543209